src/progress/pct/compile.p

```diff
@@ -537,15 +542,15 @@ PROCEDURE compileXref.
     DO i = 1 TO COMPILER:NUM-MESSAGES:
       IF COMPILER:GET-NUMBER(i) EQ 198 THEN NEXT.
       FIND ttErrors WHERE ttErrors.fileName EQ COMPILER:GET-FILE-NAME(i)
-                      AND ttErrors.rowNum   EQ COMPILER:GET-ROW(i)
-                      AND ttErrors.colNum   EQ COMPILER:GET-COLUMN(i)
+                      AND ttErrors.rowNum   EQ (IF bAbove101 THEN COMPILER:GET-ROW(i) ELSE COMPILER:GET-ERROR-ROW(i))
+                      AND ttErrors.colNum   EQ (IF bAbove101 THEN COMPILER:GET-COLUMN(i) ELSE COMPILER:GET-ERROR-COLUMN(i))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Line is 123 characters long, exceeding maximum value of 120 [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=rssw-oe-main%3Aeu.rssw.antlr.proparse.checks.LinesTooLong)